### DISCUSSION TITLE
【漏洞】rocketmq4.9.6是否有对应的springboot-starter版本

### DISCUSSION THREAD

**Discussion by kecongcong** [apache/rocketmq#6921] [General]
rocketmq4.9.6是否有对应的springboot-starter版本？如果没有的话是否有rocketmq5.1.1版本对应的springboot-starter版本

因为这几个版本存在问题

![image](https://github.com/apache/rocketmq/assets/29249422/205dabc5-9a7d-4ea9-a219-eaf4a5151ab5)

![image](https://github.com/apache/rocketmq/assets/29249422/f3193bfe-15b4-4601-8f28-8e76218ba468)

**Comment by RongtongJin** [comment]
> rocketmq4.9.6是否有对应的springboot-starter版本？如果没有的话是否有rocketmq5.1.1版本对应的springboot-starter版本
> 
> 因为这几个版本存在问题
> 
> ![image](https://user-images.githubusercontent.com/29249422/247039864-205dabc5-9a7d-4ea9-a219-eaf4a5151ab5.png)
> 
> ![image](https://user-images.githubusercontent.com/29249422/247040942-f3193bfe-15b4-4601-8f28-8e76218ba468.png)

漏洞与客户端无关

**Reply by kecongcong** [reply]
升级了服务端的话，客户端是否需要对应呢？

**Reply by RongtongJin** [reply]
是可以兼容的

**Comment by RongtongJin** [comment]
> 升级了服务端的话，客户端是否需要对应呢？
